lib/CPathfinder.h

```diff
@@ -21,6 +23,19 @@ struct TerrainTile;
 
 struct DLL_LINKAGE CGPathNode
 {
+	typedef EPathfindingLayer ELayer;
+
+	enum ENodeAction
```

Comment:
Adding another variable isn't good solution as original goal behind nodeAction is to be used by:

* isMovementAfterDestPossible. So we don't need to check rules twice.
* Cursor and path draw on client.

So I need to find solution that will result in cleaner code without tons of conditions everywhere.